x/ibc/07-tendermint/types/misbehaviour.go

```diff
@@ -53,8 +53,8 @@ func (misbehaviour Misbehaviour) String() string {
 // GetHeight returns the height at which misbehaviour occurred
 //
 // NOTE: assumes that misbehaviour headers have the same height
-func (misbehaviour Misbehaviour) GetHeight() uint64 {
-	return uint64(math.Min(float64(misbehaviour.Header1.GetHeight()), float64(misbehaviour.Header2.GetHeight())))
+func (misbehaviour Misbehaviour) GetHeight() exported.Height {
+	return misbehaviour.Header1.GetHeight()
```

Comment:
I think now that the height type changed we should actually check for the minimum `Height`, epoch, and height-wise

Reply:
or not sure if this is checked during `Misbehaviour.ValidateBasic`? since that's what's checked during message validation

Reply:
this is checked in `misbehaviour.ValidateBasic`